keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/ec2/resource/SecurityGroupHandlerTests.kt

```diff
@@ -196,7 +197,7 @@ internal object SecurityGroupHandlerTests : JUnit5Minutests {
 
     sequenceOf(
       "create" to SecurityGroupHandler::create,
-      "update" to SecurityGroupHandler::update.partially3(DiffNode.newRootNode())
+      "update" to SecurityGroupHandler::update.partially3(ResourceDiff(UpsertFixture().resource.spec))
```

Comment:
@robfletcher calling `UpsertFixture()` here seems wrong, is there a better way to reference the fixture here to get the resource?